StitchMDBKit/StitchMDBKit/LocalMongoDBService.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
```

Comment:
[q] Should these import be split into 1st, 2nd, 3rd party?

Reply:
I do not see any style guides that give any advice here, so it's up to us. We can do it that way.